Makefile

```diff
@@ -34,6 +36,10 @@ bin/grpc-client: check-go-version
 release-binary:
 	@go build -o /go/bin/dex -v -ldflags $(LD_FLAGS) $(REPO_PATH)/cmd/dex
```

Comment:
Does it make sense to reference $GOPATH here instead of hardcoding '/go/bin/dex' ?